theano/gpuarray/opt.py

```diff
@@ -1610,6 +1612,34 @@ def local_gpua_lift_abstractconv_graph(op, context_name, inputs, outputs):
     return [op(*inps)]
 
 
+@register_opt()
+@op_lifter([pool.DownsampleFactorMaxGradGrad])
+@register_opt2([pool.DownsampleFactorMaxGradGrad])
+def local_gpu_downsample_factor_max_grad_grad(op, ctx_name, inputs, outputs):
+    from theano.sandbox.cuda.opt import _check_constant_args_pool
```

Comment:
Copy the code here rather than import from the old backend.  It will make it easier when we want to delete the old one.